src/main.c

```diff
@@ -1446,32 +1276,54 @@ int main(int argc, char **argv) {
   INFO(MSG_LAUNCHER_START);
 
   zl_config_t config = read_config(argc, argv);
-
-  if (init_context(argc, argv, &config)) {
+  ConfigManager *configmgr = makeConfigManager(); /* configs,schemas,1,stderr); */
+  CFGConfig *theConfig = addConfig(configmgr,ZOWE_CONFIG_NAME);
+  cfgSetTraceStream(configmgr,stderr);
+  cfgSetTraceLevel(configmgr, configmgrTraceLevel);
+  
+  if (init_context(argc, argv, &config, configmgr)) {
     ERROR(MSG_CTX_INIT_FAILED);
     exit(EXIT_FAILURE);
   }
+
+  cfgSetConfigPath(configmgr,ZOWE_CONFIG_NAME,zl_context.yaml_file);
+
+  if (cfgLoadConfiguration(configmgr,ZOWE_CONFIG_NAME) != 0){
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "ZSS Could not load configurations\n");
+    zssStatus = ZSS_STATUS_ERROR;
+    goto out_term_stcbase;
+  }
+
+  
+
+  int schemaLoadStatus = cfgLoadSchemas(configmgr,ZOWE_CONFIG_NAME,schemas);
```

Comment:
TODO: schemas is undefined here. it should be:
https://github.com/zowe/zowe-install-packaging/blob/v2.x/master/bin/libs/configmgr.ts#L35-L38